### ISSUE TITLE
Update: Cross-site leaks Cheat Sheet

### ISSUE THREAD

**Issue by SoheilKhodayari** [OWASP/CheatSheetSeries#993]
## What is missing or needs to be updated?
In the SameSite Cookies section:

> Cookies with the appropriate SameSite attribute provide a strong defense against both XS Leaks and also CSRF attacks.

This is inaccurate because SameSite cookies do not provide complete protection against all classes of XS Leaks and CSRF attacks, but can only mitigate "some" variants of these attacks. Notable examples are client-side CSRF, or window-based XS Leaks that can abuse top-level requests to bypass the protection offered by SameSite cookies (see, e.g., [here](https://medium.com/@renwa/bypass-samesite-cookies-default-to-lax-and-get-csrf-343ba09b9f2b), and [here](https://ieeexplore.ieee.org/document/9833637)) .

Quoting from [xsleaks.dev](https://xsleaks.dev/docs/defenses/opt-in/same-site-cookies/):
> SameSite cookies are neither bulletproof nor can they fix everything.

It is also a bit ambiguous because, in my understanding, what is an "appropriate" SameSite attribute can depend on the context and functionality, and hence `SameSite=None` can also be interpreted as an appropriate SameSite attribute for some cookies while this does not provide any protection.

## How should this be resolved?

The statement should be adjusted to be more precise. Maybe something like:

SameSite cookies are a powerful **defense-in-depth** mechanism against **some** classes of XS Leaks and CSRF attacks, which can significantly reduce the attack surface, but may not completely cut them.

**Comment by mackowski** [comment]
Good Point @SoheilKhodayari! Can you please make PR with suggested change?

**Comment by SoheilKhodayari** [comment]
@mackowski of course, I'll submit a PR asap.

**Comment by mackowski** [comment]
Thanks @SoheilKhodayari !
